### PR TITLE
Alternate version of multiple-VM leaf/spine for libvirt

### PR DESCRIPTION
**Pull request by comay**
Signed-off-by: David Comay <dcomay@bloomberg.net>

This work provides a full leaf/spine/super-spine implementation using the `libvirt` Vagrant provider. It's largely the work of @mihalis68 and @tj90241 with my coming in as a relief pitcher to complete the game.

It's been tested extensively by @mihalis68 with different topologies although I've done repeated builds using simpler ones.

### CODE REVIEW

**Comment by mihalis68** [comment]
Testing from the branch before this PR was issued, I achieved 3 different 100% clean builds:
Two-pod 1h1w built clean in 30m
One-pod 3h3w3r in 68m
minimal 1 network vm 1h1w in 28m

File: virtual/Vagrantfile.libvirt [hunk 1]
```diff
@@ -105,9 +107,11 @@ Vagrant.configure('2') do |config|
 
       transit.each do |t_iface|
         args = {}.tap do |a|
+          a[:auto_config] = false
           a[:ip] = t_iface['ip'].split('/')[0]
           a[:mac] = t_iface['mac'].delete(':')
-          a[:libvirt__network_name] = Util.libvirt_prefix() + '_' + t_iface['neighbor']['name']
```
Comment (tj90241) [inline virtual/Vagrantfile.libvirt:110]: For the better, but definitely need to sent out a notice asking users to `make destroy` prior to pulling this release.

**Review by tj90241 — APPROVED** [review]
Just need to `mv -v Vagrantfile{,.vbox}; mv -v Vagrantfile{.libvirt,}` 🌶️

**Comment by comay** [comment]
> Just need to `mv -v Vagrantfile{,.vbox}; mv -v Vagrantfile{.libvirt,}` 🌶️

😹

**Pull request merged**